msteams-platform/TOC.yml

```diff
@@ -493,7 +493,7 @@
     displayName: HMAC security token
   - name: Create Incoming Webhooks
     href: webhooks-and-connectors/how-to/add-incoming-webhook.md
-  - name: Create Office 365 Connectors
+  - name: Create Connectors for Microsoft 365 Groups
```

Comment:
Let's de-capitalize "Connector" here (since it's not the first word of the TOC entry and isn't otherwise capitalized in the phrase "connectors for Microsoft 365 Groups")

Reply:
Updated the TOC entry.